Arashayo/TabletopHaven
# Add user shelf view

## Related Issue
https://app.clickup.com/t/86bxdbgpw

## Changes list
Added your shelf view
Added profile picture to database
Added profile picture fetching
Added smaller game cards variant for cleaner profile view (up to you if we want to leave them)
Fixed a small issue with adding game to owned

## Testing instructions

1. run `node backend/src/createTables.js`
2. run `node backend/src/fillGames.js`
3. log in
4. add some games to owned/wishlist/favorites
5. check the your shelf view
6. tell me if you like the smaller game cards there

## Screenshots (if appropriate):
